frontend-react/src/components/Table/Table.test.tsx

```diff
@@ -28,57 +81,104 @@ const clickFilterButton = () => {
+    );
+};
+
 describe("Table, basic tests", () => {
-    beforeEach(() => renderWithRouter(<TestTable />));
     test("Column names render", () => {
-        const headerOne = screen.getByText("Column One");
-        const headerTwo = screen.getByText("Column Two");
+        renderWithRouter(<SimpleTable />);
+        const idHeader = screen.getByText("Id");
+        const itemHeader = screen.getByText("Item");
 
-        expect(headerOne).toBeInTheDocument();
-        expect(headerTwo).toBeInTheDocument();
+        expect(idHeader).toBeInTheDocument();
+        expect(itemHeader).toBeInTheDocument();
     });
 
     test("Row values render", () => {
-        const rowOne = screen.getByText("value two");
-        const rowTwo = screen.getByText("value two again");
-
-        expect(rowOne).toBeInTheDocument();
-        expect(rowTwo).toBeInTheDocument();
-    });
-
-    test("Rows render data in configured order", () => {
-        const allRows = screen.getAllByRole("row");
-        expect(allRows[2].firstChild).toHaveTextContent("value two");
-    });
-
-    test("Elements with no mapped column aren't rendered", () => {
-        expect(screen.queryByText("value three")).not.toBeInTheDocument();
+        renderWithRouter(<SimpleTable />);
+        expect(screen.getAllByRole("columnheader").length).toEqual(7);
+        expect(screen.getAllByRole("row").length).toEqual(11); // +1 for header row
+        expect(screen.getByText("Item 1")).toBeInTheDocument();
     });
 
     test("Link columns are rendered as links", () => {
-        const linkInCell = screen.getByText("value two");
-        /* This looks ugly but it's ensuring the VALUE at linkAttr
-         * (or dataAttr) is plugged into the href.
-         *
-         * "/test/value two" is the outcome with the test data
-         * but the attributes we plug in don't have spaces to account
-         * for */
+        renderWithRouter(<SimpleTable />);
+        const linkInCell = screen.getByText("UUID-1");
         expect(linkInCell).toContainHTML(
-            '<a class="usa-link" href="/test/value two">value two</a>'
+            '<a class="usa-link" href="/base/UUID-1">UUID-1</a>'
         );
     });
 
     test("Map columns use mapped value", () => {
-        const mappedElement = screen.getAllByRole("row")[1].children[3];
-        expect(mappedElement).toHaveTextContent("mapped value");
+        renderWithRouter(<SimpleTable />);
+        expect(screen.getByText("Mapped Item")).toBeInTheDocument();
     });
 
     test("Transform columns use transformed value", () => {
-        const transformedElements = screen.getAllByText("transformed");
-        expect(transformedElements).toHaveLength(2);
+        renderWithRouter(<SimpleTable />);
+        expect(screen.getByText("Transformed Value")).toBeInTheDocument();
     });
 });
 
+describe("Sorting integration", () => {
+    test("(Locally) Sorting swaps on header click", () => {
```

Comment:
When trying to assert the content order change, I ran into some issues with the test renderer; clearly it _works_ in browser, but the test renderer wasn't change memoized rows. Help would be much appreciated!